contracts/contracts/Utilities.sol

```diff
@@ -0,0 +1,37 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity >=0.8.0 <0.9.0;
+pragma abicoder v2;
+
+contract Utilities {
+  struct FunctionCall {
+    address contractAddress;
+    bytes encodedFunction;
+  }
+
+  struct FunctionResult {
+    bool success;
+    bytes returnValue;
+  }
+
+  function performSequence(
```

Comment:
Would be good to have [NatSpec](https://docs.soliditylang.org/en/v0.8.11/natspec-format.html) comments/docs for this function and contract. Can do this as follow up.

Reply:
Done